Automattic/wp-calypso
# Remove duplicate views: Update experiment name

Part of https://github.com/Automattic/wp-calypso/issues/97398

## Proposed Changes

We used a fake experiment (22124-explat-experiment) in https://github.com/Automattic/wp-calypso/pull/97231 so we could start the "Remove duplicate views" project without being blocked by the lack of the experiment properly designed in ExPlat.

We now have a real experiment (22129-explat-experiment), so this PR replaces the previous experiment name with the right one.

## Testing Instructions

- Go to the final experiment page: 22129-explat-experiment
- Assign yourself to the `treatment` variation using the bookmarklets
- Use the Calypso live link below
- Go to any of the following paths for a site with the default admin interface
  - `/posts/:site`
  - `/settings/taxonomies/category/:site`
  - `/settings/taxonomies/post_tag/:site`
  - `/types/jetpack-testimonial/:site`
  - `/types/jetpack-portfolio/:site`
- Make sure you're redirected to WP Admin

## Pre-merge Checklist
